README.rst

```diff
@@ -5,19 +5,32 @@ UMI-tools was published in `Genome Research <http://genome.cshlp.org/content/ear
 Tools for dealing with Unique Molecular Identifiers
 ====================================================
 
-This repository contains tools for dealing with Unique Molecular Identifiers (UMIs)/Random Molecular Tags (RMTs). Currently there are three tools:
+This repository contains tools for dealing with Unique Molecular
+Identifiers (UMIs)/Random Molecular Tags (RMTs). Currently there are
+five tools
 
 * extract:
    **Flexible removal of UMI sequences from fastq reads.**
       UMIs are removed and appended to the read name. Any other barcode, for example a library barcode, is left on the read.  
 
+* group: 
```

Comment:
Good point. `README.rst` should also describe `count` and `count_tab` since these commands are also now available.